src/cljs/main/broadfcui/common/gcs_file_preview.cljs

```diff
@@ -59,7 +62,14 @@
                                :target "_blank"}
                            "Open" icons/external-link-icon]
                           [:span {:style {:fontStyle "italic" :color (:text-light style/colors)}}
-                           " (right-click to download)"]])))
+                           " (right-click to download)"]]))
+                      (when (> data-size 100000000)
+                        (react/create-element
+                         [:div {:style {:marginTop "1em" :marginBottom "1em"}}
+                          [:div {} "Downloading large files through the browser may not be successful. Instead use this gsutil"]
+                          [:div {:style {:marginBottom ".5em"}} "command replacing [DESTINATION] with the local file path you wish to download to."]
+                          (style-guide/create-code-sample
```

Comment:
I don't like importing the style guide for this. Let's extract this function and put it in style.cljs.